templates/case/_identifier.html.twig

```diff
@@ -0,0 +1,16 @@
+{% trans_default_domain 'case' %}
+
+{% set id = attribute(case, id_property) %}
+
+<div class="form-group col-md-2">
+    <label>{% trans %}Identifier{% endtrans %}</label>
+    <input type="text" class="form-control" value="{{ id.identifier }}" readonly>
+</div>
+<div class="form-group col-md-3">
+    <label>&nbsp;</label>
```

Comment:
Remove this empty `label` element.

Reply:
It was placed to ensure correct alignment between the input fields and the validate button. Added label with `class="invisible"` instead.